src/plugins/crypto/README.md

```diff
@@ -109,34 +110,35 @@ Set the CMake variables `OPENSSL_INCLUDE_DIR` and `OPENSSL_LIBRARIES` to your de
 
 The crypto plugin will encrypt and decrypt values using AES-256 in CBC mode.
 
-The key derivation is still WIP.
+The GPG binary is required for handling the cryptographic keys.
 
-### Planned Features ###
+At the moment the plugins will only run in a UNIX/Linux-like system.
 
-- Encryption of values
-- Decryption of values
-- Key derivation by metadata (password provided within a meta-key)
-- Key derivation by using a specified key-file (like the SSH client does)
-- Key derivation by utilizing the pgp-agent
+## Configuration ##
 
-The encryption and decryption of values is a straight forward process, once the key and IV are supplied.
-The key-derivation process in a library is a bit tricky.
+### GPG Configuration ###
 
-The crypto plugin itself can hold configuration data about the order of the applied key derivation functions.
-For example, if a password is provided by a meta-key, it will be used, otherwise we look for a key file.
-If no key file has been configured, we try to trigger the PGP agent, etc.
+The path to the gpg binary can be specified in
 
-The following example configuration illustrates this concept:
+	/crypto/gpg
 
-	system/elektra/crypto/config/key-derivation/#0 = meta
-	system/elektra/crypto/config/key-derivation/#1 = file
-	system/elektra/crypto/config/key-derivation/#2 = agent
-	system/elektra/crypto/config/key-file/path/#0 = ~/.elektra/id_aes
-	system/elektra/crypto/config/key-file/path/#1 = /etc/elektra/id_aes
+The GPG recipient keys can be specified as `/crypto/key` directly.
+If you want to use more than one key, just enumerate like:
 
-Only keys marked with a certain meta-key will be considered for encryption/decryption.
+	/crypto/key/#0
```

Comment:
Maybe `/gpg/key` is even better, because it is the main configuration, it is also nice if it is short.